css/src/components/persona.scss

```diff
@@ -0,0 +1,82 @@
+$persona-font-size-xs: $font-size-9 !default;
+$persona-font-size-sm: $font-size-8 !default;
+$persona-font-size-md: $font-size-7 !default;
+$persona-font-size-lg: $font-size-6 !default;
```

Comment:
The difference between large and extra large is 2 pixels. Let's get rid of xl.

Reply:
Please read the thread above about sizes.

Reply:
Updated